wordpress-mobile/WordPress-Android
# Update context menus in Pages list Action

## Closes
#19356 

## Description
This PR
1.  ↑ Updates:  the Context menu of the pages list actions with divider and new icons 
2. ↑ Updates: the order of the pages list actions according to the design 

## 📷  Screenshots
|  Published by Admin | Published by Contributor  | Scheduled | Draft  | Trash   |
|---|---|---|---|---|
|  <img width="578" alt="Screenshot 2023-11-02 at 1 38 38 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/2593a28d-29a9-4602-a4c1-3f7860363ed5"> |<img width="586" alt="Screenshot 2023-11-02 at 1 38 50 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/39cc53af-3880-4125-8f0b-ac4919817262"> |<img width="578" alt="Screenshot 2023-11-02 at 1 40 09 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/4a127a7e-1da7-4e30-8f76-64a3f480b500"> | <img width="603" alt="Screenshot 2023-11-02 at 1 39 02 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/54801962-080d-48fb-970e-34744e6f717e">|<img width="573" alt="Screenshot 2023-11-02 at 1 39 12 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/d3622f99-ce8e-42fd-ae6e-aa3b0fada408"> |

📓 There is no changes to functionalities of the pages list item actions  

## To test:

### Prereqs
Have a few accounts with editor, contributor, and admin capabilities

### Test
1. Create a fews Pages as a contributor - drafts, scheduled, and non-scheduled (submit these)
2. Create a few Pages as editor/admin
3. Install the JP app
4. Launch and sign in
5. Navigate to My Site > Pages
6. Tap on the "more" icon on a pages row in the Published tab
7. ✅ Verify the new icon groupings are shown, the icons and copy are correct
8. Repeat the last two steps for each tab

### Test - Repeat for WP app 

## ⚠️  Merge Instructions
1. Wait for the https://github.com/wordpress-mobile/WordPress-Android/pull/19498 to merge
2. Make sure that the base branch changes to `feature/ui-modernization-posts-and-pages` or to `trunk` - depending on the merge of feature branch PR 
3. Remove Do not merge label 
4. Merge as normal

## Regression Notes
1. Potential unintended areas of impact
Icons are not shown as intended and not grouped as intended

5. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

6. What automated tests I added (or what prevented me from doing so)
Manual testing 

PR submission checklist:

UI Changes testing checklist:
